packages/shared/src/components/CustomFeedOptionsMenu.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { type ReactElement } from 'react';
+import classNames from 'classnames';
+import { ContextMenuIds } from '../hooks/constants';
+import type { MenuItemProps } from './fields/ContextMenu';
+import ContextMenu from './fields/ContextMenu';
+import { HashtagIcon, MenuIcon as DotsIcon, ShareIcon } from './icons';
+import { MenuIcon } from './MenuIcon';
+import useContextMenu from '../hooks/useContextMenu';
+import { Button, ButtonSize, ButtonVariant } from './buttons/Button';
+import {
+  useShareOrCopyLink,
+  type UseShareOrCopyLinkProps,
+} from '../hooks/useShareOrCopyLink';
+
+type CustomFeedOptionsMenuProps = {
+  className?: string;
+  shareProps: UseShareOrCopyLinkProps;
+};
+
+const CustomFeedOptionsMenu = ({
+  className,
+  shareProps,
+}: CustomFeedOptionsMenuProps): ReactElement => {
+  const [, onShareOrCopyLink] = useShareOrCopyLink(shareProps);
+  const { isOpen, onMenuClick } = useContextMenu({
+    id: ContextMenuIds.CustomFeedContext,
+  });
+
+  const options: MenuItemProps[] = [
+    {
+      icon: <MenuIcon Icon={ShareIcon} />,
+      label: 'Share',
+      action: () => onShareOrCopyLink(),
+    },
+    {
+      icon: <MenuIcon Icon={HashtagIcon} />,
+      label: 'Add to custom feed',
+      action: () => {
+        // TODO: Implement modal || funnel to upgrade to plus
+      },
+    },
+  ];
+
+  return (
+    <>
+      <Button
+        className={classNames('!border-none !px-1.5', className)}
```

Comment:
Don't think you need the border-none and px right?
If you pass icon as icon={} it will render as iconOnly component.

Reply:
Not sure what I goofed up initially when I tried just passing the icon to the button icon prop, but it positioned it to the side 🥴 

However, seems to be no issues now, so changed it ✅